### DISCUSSION TITLE
Where can I find information on the template styles available?

### DISCUSSION THREAD

**Discussion by anthonyomorada** [2anki/server#1555] [Q&A]
Hi, where can I find the templates for the cards being created?
Sometimes, I change the text format in Notion by bolding, italicizing, changing colors, etc. How can I preserve this when creating the Anki flashcards? 

![image](https://github.com/2anki/server/assets/10801936/a090d883-f1a6-4d3c-bf21-946de25930ea)

**Comment by aalemayhu** [comment]
@anthonyomorada there is now a dedicated templates section https://2anki.net/templates/
